truss/templates/tgi/tgi.Dockerfile.jinja

```diff
@@ -28,10 +23,26 @@ RUN python3 /cache_warmer.py {{file}} {{repo}} {% if hf_dir.revision != None %}{
         {% endfor %}
 {%- endif %}
 
+
+FROM ghcr.io/huggingface/text-generation-inference:0.9.4 as tgi
```

Comment:
@aspctu is it okay to bump the version here?

Reply:
also no need for the `as tgi`

Reply:
@bolasim yes